defaults/main.yml

```diff
@@ -22,7 +22,6 @@ dell_dsu_gpg_keys:
 dell_dsu_arguments_base: "--non-interactive --apply-upgrades"
 dell_dsu_prerequisites:
   - usbutils
-  - compat-libstdc++-33.i686
```

Comment:
I think that you have to remove it only for centos8-stream. You can gather the fact and apply compat-libstdc++-33.i686 only in case the OS is not centos8-stream. Otherwise, there is a risk to break things for Centos7. I did not checked deeply but you have to confirm first.

Reply:
We can check your changes in one of our devel nodes that use Centos7.

Reply:
If you could test removing the whole prequisite task and try install dsu, that would be great.

Reply:
I will rebuilt a new node and do the test. I will let you know.

Reply:
If it woks, then you can push a new commit.

Reply:
I have rhel8 systems where I recently applied updates, they have neither of the 3 packages installed, so I can confirm it for rhel8, for rhel7 I can not say it for sure about all three packages, usbutils is installed on all systems where I use dsu.

Reply:
I will check for Centos7.

Reply:
These changes worked for me.

Reply:
Looks good, thank you.